cognite/neat/_graph/queries/_base.py

```diff
@@ -219,7 +218,11 @@ def describe(
                 property_ = RDF.type
                 renamed_property_ = property_
 
-            if isinstance(object_, URIRef):
+            value: str | URIRef
+            if property_ in SPACE_PROPERTIES and isinstance(object_, URIRef):
+                # These properties contain the space in the Namespace.
+                value = object_
```

Comment:
For start and end node we cannot remove the namespace as that contain which space to write to.